project/Build.scala

```diff
@@ -276,22 +276,20 @@ object DottyBuild extends Build {
       // Override run to be able to run compiled classfiles
       dotr := {
         val args: Seq[String] = spaceDelimited("<arg>").parsed
-        val java: String = Process("which" :: "java" :: Nil) !!
-        val scalaLib = (dependencyClasspath in Runtime, packageAll)
-          .map { (attList, _) =>
-            attList
-              .map(_.data.getAbsolutePath)
-              .find(_.contains("scala-library"))
-              .toList.mkString(":")
-          }.value
+        val java: String = Process("which" :: "java" :: Nil).!!
+        val attList = (dependencyClasspath in Runtime).value
+        val scalaLib = attList
+          .map(_.data.getAbsolutePath)
+          .find(_.contains("scala-library"))
+          .toList.mkString(":")
```

Comment:
You are dropping the dependency of this task on `packageAll` here. I guess this was not intentional. You should add
```scala
val _ = packageAll.value
```